CHANGELOG.md

```diff
@@ -0,0 +1,6 @@
+# Change Log
+
+All notable changes to this project will be documented in this file
+automatically by Versionist. DO NOT EDIT THIS FILE MANUALLY!
+This project adheres to [Semantic Versioning](http://semver.org/).
+
```

Comment:
This'll actually also need an initial version, as `versionist` needs to be able to retrieve the difference between versions to determine the new semver version.

See [here](https://github.com/resin-io/process/blob/master/process/Commit_and_PR_Guidelines.md) in the 'Prerequisites' section.